pkg/handlers/service_logs.go

```diff
@@ -102,24 +94,49 @@ func (b *ServiceLogBuilder) Build(firing bool, alert *template.Alert) (*ServiceL
 		}
 	}
 
-	return b.wrappedBuilder.Summary(summary).Description(description).Build()
+	// Handle DocReferences
+	if len(b.references) > 0 {
+		for _, ref := range b.references {
+			docReferences = append(docReferences, string(ref))
+		}
+	}
+
+	// Directly assign the docReferences slice
+	logEntry, err := b.wrappedBuilder.
+		Summary(summary).
+		Description(description).
+		DocReferences(docReferences...).
+		Build()
+
+	return logEntry, err
 }
 
-func NewTestServiceLog(summary, desc, clusterUUID string, severity v1alpha1.NotificationSeverity, logType string) *ServiceLog {
-	sl, _ := slv1.NewLogEntry().
+func NewTestServiceLog(summary, desc, clusterUUID string, severity v1alpha1.NotificationSeverity, logType string, references []v1alpha1.NotificationReferenceType) *ServiceLog {
```

Comment:
Should be better to put this function in test.go?

Reply:
This function was initially created by Nicolas and I didn't want to alter the previous structure/style but focus on updating the logic to separate the `references` from `description` instead.  But yeah, moving `NewTestServiceLog` function to `service_logs.go` is better considering it's solely used for testing purposes.